src/main/java/org/opentripplanner/model/OtpTransitService.java

```diff
@@ -77,4 +77,9 @@ public interface OtpTransitService {
   Collection<Trip> getAllTrips();
 
   Collection<FlexTrip> getAllFlexTrips();
+
+  /**
+   * @return if transit service has any active services
```

Comment:
It would be nice here to document what "active" means, is it related to the configured 'transitServiceStart' and 'transitServiceEnd'?

Reply:
Clarified it